application/libraries/admin/quexmlpdf.php

```diff
@@ -3806,6 +3806,16 @@ protected function newPage($init = false)
             $this->SetHeaderMargin(0);
             $this->SetFooterMargin(0);
 
+            $oTemplate = Template::model()->getInstance();
+            $sLogoFileName = $oTemplate->filesPath . Yii::app()->getConfig('pdflogofile');
+            if (file_exists($sLogoFileName)) {
+                $result = LSYii_ImageValidator::validateImage($sLogoFileName);
+                if (isset($result['check']) && $result['check'] == true) {
+                    $sLogo = "@" . file_get_contents($sLogoFileName);
```

Comment:
You probably want to check if `file_get_contents` returned anything valid.

Reply:
Done in `$result = LSYii_ImageValidator::validateImage($sLogoFileName);` :)

Reply:
No? It can return `false`, which should be checked.